load/resources/queries/08-add-qudt-terms.ru

```diff
@@ -0,0 +1,40 @@
+PREFIX qudt: <http://qudt.org/schema/qudt/>
```

Comment:
I haven't checked these, but I'll check the result and hope it's ok and useful to @ar-chad